src/components/Playbutton.jsx

```diff
@@ -1,5 +1,11 @@
-export const Playbutton = () => (
-  <div className="play-button">
+export const Playbutton = ({ handlePlay }) => (
+  <div
```

Comment:
Needs an aria-label. 
Should also be a button. You'd save yourself the hassle of the onKeyDown, tabIndex & role.